neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/impl/BaseNeo4jSchemaDefiner.java

```diff
@@ -82,6 +87,35 @@ public String toString() {
 
 	private static final Log log = LoggerFactory.make( MethodHandles.lookup() );
 
+	protected void createIndexes(DatastoreProvider provider, Database database) {
+		List<Neo4jIndexSpec> indexes = new ArrayList<>();
```

Comment:
Maybe we can refactor the **indexes** local variable as a Class field, like in MongoDBSchemaDefiner class.
You could fill the list on validateMapping and create the indexes on initializeSchema, like in MongoDBSchemaDefiner. WDYT?